test/unit/assorted/max_staleness.spec.test.js

```diff
@@ -45,13 +45,12 @@ describe('Max Staleness (spec)', function () {
   });
 
   const specTests = collectStalenessTests(maxStalenessDir);
-  Object.keys(specTests).forEach(specTestName => {
+  for (const [specTestName, test] of Object.entries(specTests)) {
```

Comment:
I was debugging these tests for a leak and refactored this as a drive-by improvement.